fix(laravel): don't edit database config in `.env` when there's no database (#7584)
drud/ddev: docs/content/users/quickstart.md

    ```bash
    mkdir my-laravel-site && cd my-laravel-site

>     # If you're using DDEV v1.24.8+, you don't need --disable-settings-management=true
>     ddev config --project-type=laravel --docroot=public --omit-containers=db --disable-settings-management=true

Reviewer: ```suggestion
    # If you're using  an older version of DDEV (<=v1.24.7), you need to add `--disable-settings-management=true`
ddev config --project-type=laravel --docroot=public --omit-containers=db
```

Reviewer 2: I thought about changing the default later, but I agree, let's do it now.

    ```bash
    mkdir my-laravel-site && cd my-laravel-site

>     # If you're using an older version of DDEV (<=v1.24.7),

Reviewer 3: I'm reluctant to add version specs to the docs that get out of date so quickly. I wonder if there's another technique to work on this problem.

Reviewer 2: Another way is not to update the quickstart, as we did with `ddev composer create-project`.
Wait a few months and only do it later.

Reviewer: Thank you.

    ```bash
    mkdir my-laravel-site && cd my-laravel-site

>     # If you're using  an older version of DDEV (<=v1.24.7),

Reviewer 2: ```suggestion
    # If you're using an older version of DDEV (<=v1.24.7),
```

    To switch an existing Laravel project to SQLite:

    ```bash

>     # If you're using  an older version of DDEV (<=v1.24.7),

Reviewer 2: ```suggestion
    # If you're using an older version of DDEV (<=v1.24.7),
```

>     you need to add `--disable-settings-management=true`

Reviewer 2: ```suggestion
    # you need to add `--disable-settings-management=true`
```
